oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -43,6 +43,14 @@
 #define SPML_UCX_PUT_DEBUG    0
 #endif
 
+#define SSHMEM_MASK_SIZE (sizeof(((mca_spml_ucx_ctx_t*)NULL)->put_op_mask[0]))
```

Comment:
1. pls indent macros on column
1. can typedef mca_spml_ucx_bitmap_elem_t uint64_t
1. pls use MCA_SPML_UCX prefix, not SSHMEM
1. pls check if we can reuse any macros from OPAL

Reply:
implemented item 4 - reused opal bitmap infra